api/src/main/java/org/openmrs/api/impl/PersonServiceImpl.java

```diff
@@ -488,18 +488,11 @@ public Relationship saveRelationship(Relationship relationship) throws APIExcept
 	 */
 	@Override
 	public Relationship voidRelationship(Relationship relationship, String voidReason) throws APIException {
-		if (relationship.getVoided()) {
-			return relationship;
-		}
-		
-		relationship.setVoided(true);
-		if (relationship.getVoidedBy() == null) {
-			relationship.setVoidedBy(Context.getAuthenticatedUser());
-		}
-		if (voidReason != null) {
-			relationship.setVoidReason(voidReason);
+		if(relationship == null) {
+			
```

Comment:
Do you need this extra space ? at lines 492 and 494

Reply:
@tendomart  https://github.com/openmrs/openmrs-core/pull/3890/commits/92a512760f8459668a5af3145777701bb776bd86 hope this is right